net.sf.VICE.yml

```
@@ -10,54 +10,144 @@ finish-args:
   - --socket=pulseaudio
   - --socket=x11
   - --metadata=X-DConf=migrate-path=/net/sf/VICE/
+add-extensions: 
+  - org.freedesktop.Platform.ffmpeg-full: 
+    directory: lib/ffmpeg
+    version: 21.08
+    add-ld-path: .
```

```suggestion
    org.freedesktop.Platform.ffmpeg-full: 
        directory: lib/ffmpeg
        version: '21.08'
        add-ld-path: .
```

This is the proper syntax.

It even complain with `(flatpak-builder:2): Json-WARNING **: 02:44:55.926: Failed to deserialize "add-extensions" property of type "gpointer" for an object of type "BuilderManifest"`

Also `(flatpak-builder:2): flatpak-builder-WARNING **: 02:44:55.925: 16:14: '21.08' will be parsed as a number by many YAML parsers`

(YAML is amazing)

---

Just ignore this, see suggestio below.